### PR TITLE
test(featctl): add integration test of apply entity

### PR DESCRIPTION
**Pull request by lianxmfor**
this PR is add integration test of apply entity

### CODE REVIEW

File: featctl/test/test_apply_entity.sh [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+EOF
+
+featctl apply -f "$TMPFILE"
+
+expected='Name,Length,Description
+user,8,description
+'
+actual=$(featctl list entity -o csv)
+ignore_time() { cut -d ',' -f 1-3 <<<"$1"; }
+assert_eq "featctl list entity" "$(ignore_time "$expected" | sort)" "$(ignore_time "$actual" | sort)"
```
[the remark this answers is not in the document]
Comment (YikSanChan) [reply featctl/test/test_apply_entity.sh:43]: same to below

**Review by YikSanChan — APPROVED** [review]
lgtm

**Comment by YikSanChan** [comment]
plz add more tests to cover
```yaml
...
---
...
---
...
```

File: featctl/test/test_apply_entity.sh [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    db-value-type: int
+    description: 'description'
+  - name: gender
+    db-value-type: int
+    description: 'description'
+EOF
+
+featctl apply -f "$TMPFILE"
+
+expected='Name,Length,Description
```
[the remark this answers is not in the document]
Comment (lianxmfor) [reply featctl/test/test_apply_entity.sh]: fixed

File: featctl/test/test_apply_entity.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (lianxmfor) [reply featctl/test/test_apply_entity.sh]: all done

**Comment by lianxmfor** [comment]
> plz add more tests to cover
> 
> ```yaml
> ...
> ---
> ...
> ---
> ...
> ```

added more tests

**Pull request merged**